packages/backend/src/helpers/auth.ts

```diff
@@ -71,3 +71,22 @@ export async function getOrCreateUser(email: string): Promise<User> {
 
   return user
 }
+
+export async function validateAdminTokenAndGetUser(
+  token: string,
+): Promise<User | null> {
+  // NOTE: we use a different key to prevent a vuln where an end user can send
+  // their auth cookie in the `x-plumber-admin-token` header value to gain
+  // admin access.
+  const { userEmail } = jwt.verify(token, appConfig.adminJwtSecretKey) as {
```

Comment:
might be more sane to verify maxAge here, if we somehow forget to set on admin portal side

Reply:
good call, done!